charts/ziti-host/README.md

```diff
@@ -52,6 +52,7 @@ When you don't want to use the default key name `persisted-identity` you can def
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| additionalVolumes | list | `[]` | additional volumes, see values.yaml for example |
```

Comment:
The README.md gets generated by CI from the .tmpl file. I think we can leave this in because it'll overwrite with the comment from values.yaml.